pixart.js

```diff
@@ -1 +1,43 @@
+console.log('script.js loaded...');
+//globals
+var pixelForm = document.getElementById('form');
+var colorBox = document.querySelector('.brush');
+var colorInput = document.getElementById('color-field');
+var colorButton = document.getElementById('set-color');
 
+//create 20 div's with class .square and append them to document.body
+for (var i = 2; i < 8000; i++) {
+  var extraDiv = document.createElement('div');
+  extraDiv.className = 'square'
+  document.body.appendChild(extraDiv);
+  extraDiv.addEventListener('mouseover', function() {
+    this.style.backgroundColor = colorInput.value;
+  })
+}
+
+
+//EVENT HANDLERS
+//change brush box color when colorButton is clicked...
+colorButton.addEventListener('click', buttonTrigger)
+colorInput.addEventListener('keydown', keyTrigger)
+
+
+//event handler functions
+function keyTrigger(e) {
+  if(event.code == 'Enter') {
```

Comment:
good catch on the `keydown` "enter" event! However, there is duplicate code here as well as below in `buttonTrigger `function. Could you think of a way to simplify it? Maybe having one function that just assigns the color?

```
function assignColor(){
  event.preventDefault();
  colorBox.style.backgroundColor = colorInput.value;
}
```

```
function keyTrigger() {
  if(event.code == 'Enter') {
    event.preventDefault();
    assignColor();
  }
}
```

Something along those lines :+1: